### ISSUE TITLE
Update GEFS to output numerical icing severity and remove ICIP

### ISSUE THREAD

**Issue by HuiyaChuang-NOAA** [NOAA-EMC/UPP#893]
@YaliMao-NOAA  will update UPP to output numerical icing severity so that we can compute probabilistic icing severity to meet ICAO 2027 milestone.  She already emailed @AndrewBenjamin-NOAA to request a new Grib2 ID for this new variable

Including @bingfu-NOAA and @WenMeng-NOAA for awareness

**Comment by AndrewBenjamin-NOAA** [comment]
Continuous Icing Severity has been added to [Table 4.2-0-19](https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-19.shtml):

240 | Continuous Icing Severity | non-dim | ICESEVCON

**Comment by HuiyaChuang-NOAA** [comment]
> Continuous Icing Severity has been added to [Table 4.2-0-19](https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-19.shtml):
> 
> 240 | Continuous Icing Severity | non-dim | ICESEVCON

Thank you

**Comment by HuiyaChuang-NOAA** [comment]
@bingfu-NOAA @YaliMao-NOAA 
We also may need to add GTG output to a subset or all (if possible) of GEFS members to fulfill ICAO 2027 milestone.
Initially, Yali was going to add generation of GTG in her separated out, stand-alone WAFS package but this option is
no longer valid

**Comment by WenMeng-NOAA** [comment]
@HuiyaChuang-NOAA @YaliMao-NOAA @BinLiu-NOAA With aviation products separation from GFS/GEFS implementations, can we close this issue and address them in WAFS-related implementation?

**Comment by HuiyaChuang-NOAA** [comment]
> [@HuiyaChuang-NOAA](https://github.com/HuiyaChuang-NOAA) [@YaliMao-NOAA](https://github.com/YaliMao-NOAA) [@BinLiu-NOAA](https://github.com/BinLiu-NOAA) With aviation products separation from GFS/GEFS implementations, can we close this issue and address them in WAFS-related implementation?

@WenMeng-NOAA Yali just sent a request to GEFS group for saving GEFS model history files to com/ to support next upgrade of WAFS to produce probabilistic aviation output.  I'd like to suggest waiting until receipt of this approval to close this ticket.  Thank you!

**Comment by WenMeng-NOAA** [comment]
Sorry for the typo. tagging @bingfu-NOAA in stead of @BinLiu-NOAA.

**Comment by bingfu-NOAA** [comment]
@WenMeng-NOAA @HuiyaChuang-NOAA @YaliMao-NOAA Yes, 5T per day is a big number, we are fine to update global-workflow to retain the history file, but we are not sure if NCO will allow us to use another 5T per day (current GEFSv12 is using 14T per day). We will try to find the answer from NCO.
